FrontEnd/src/App.js

```diff
@@ -26,7 +26,7 @@ function App() {
         <Header isAuthorized={true}></Header>
         <Routes>
           <Route path="/" element={<MainPage />} />
-          <Route path="/authorization" element={<MainPageRectangle />} />
+          <Route path="/authorization" element={<AuthorizationPage />} />
           <Route path="/SignUpPage" element={<SignUpPage />} />
           <Route path="/PrivacyPolicyPage" element={<PrivacyPolicy />} />
```

Comment:
There is no need to add `Page` to the path. Usually, for URLs, we divide words using a dash. 
Example `privacy-policy`

Reply:
Fixed